pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -749,6 +749,11 @@ export class CreateVmAction extends React.Component {
         this.close = this.close.bind(this);
     }
 
+    componentWillMount() {
+        cockpit.script('virt-install --version', null, { err: 'message' })
```

Comment:
Please use `cockpit.spawn()` for such things to at least avoid a shell. Also, `["which", "virt-install"]` is much cheaper as it doesn't involve starting a Python interpreter.